apps/quickbooks_online/utils.py

```
@@ -1411,3 +1411,32 @@
         return entity_map
+
+    def get_exported_entry(self, resource_type: str, export_id: str):
+        """
+        Retrieve a specific resource by internal ID.
+
+        Args:
+            resource_type (str): The type of resource to fetch.
+            export_id (str): The internal ID of the resource.
+        """
+        module = getattr(self.connection, resource_type)
+        response = getattr(module, 'get_by_id')(export_id)
```

Ruff (0.7.0)

Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)